server/startup/migrations/v150.js

```diff
@@ -0,0 +1,37 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Migrations } from '../../../app/migrations/server';
+import { Users } from '../../../app/models/server';
+import { MAX_RESUME_LOGIN_TOKENS } from '../../lib/accounts';
+
+Migrations.add({
+	version: 150,
+	async up() {
```

Comment:
I'm not sure it makes much difference having this as `async`, I don't it waits the promise to end to go to next migration (when we have one)..

in this case it might be good as this query _could_ be slow though.